bundles/org.openhab.binding.yeelight/README.md

```diff
@@ -4,11 +4,16 @@ This binding integrates the [Yeelight Lighting Product](https://www.yeelight.com
 
 ## Supported Things
 
-- [Yeelight LED White Bulb](https://www.yeelight.com/zh_CN/product/wifi-led-w) (Thing type `dolphin`)
-- [Yeelight LED Color Bulb](https://www.yeelight.com/zh_CN/product/wifi-led-c) (Thing type `wonder`)
-- [Yeelight LED Color Stripe](https://www.yeelight.com/zh_CN/product/pitaya) (Thing type `stripe`)
-- [Yeelight LED Ceiling Light](https://www.yeelight.com/en_US/product/luna) (Thing type `ceiling`)
-
+| Device                                                                                 | Thing ID   | Device model                      |
+|----------------------------------------------------------------------------------------|------------|-----------------------------------|
+| [Yeelight LED Ceiling Light](https://us.yeelight.com/category/ceiling-light/)          | `ceiling`  | `ceiling`, `ceiling3`             |
+| [Yeelight LED Ceiling Light](https://us.yeelight.com/category/ceiling-light/)          | `ceiling1` | `ceiling1`, `ceiling11`, `ceil26` |
+| [Yeelight LED Ceiling Light](https://us.yeelight.com/category/ceiling-light/)          | `ceiling4` | `ceiling4`                        |
+| [Yeelight LED Color Bulb](https://us.yeelight.com/category/smart-bulb/)                | `wonder`   | `color`, `color4`                 |
+| [Yeelight LED White Bulb](https://us.yeelight.com/category/smart-bulb/)                | `dolhin`   | `mono`                            |
+| [Yeelight LED White Bulb v2](https://us.yeelight.com/category/smart-bulb/)             | `ct_bulb`  | `ct_bulb`                         |
+| [Yeelight LED Color Stripe](https://us.yeelight.com/shop/yeelight-led-lightstrip-pro/) | `stripe`   | `stripe`, `strip6`                |
```

Comment:
The link leads to a blank page. Maybe rather use https://us.yeelight.com/category/led-strip/ ?